spec/requests/create_bookmark_from_email_spec.rb

```diff
@@ -13,12 +13,23 @@
     expect(@user.topics.first.bookmarks).to match_array [Bookmark.find_by(url: "www.google.com")]
     expect(response.status).to eq(200) 
   end
-  xit "creates a bookmark if topic already exists" do
+  it "creates a bookmark if topic already exists" do
+    topic = create(:topic, title: "animals", user: @user)
+    post '/bookmarks',{sender: "test@example.com", subject: "animals", "body-plain": "www.google.com"}
+    expect(@user.topics.first.bookmarks.first).to belong_to(:topic)
```

Comment:
You were right, the test passes when I commented out line 17. I changed it
to : expect(Bookmark.find_by_url("www.google.com").topic).to eq(topic)
and added :  expect(@user.topics.count).to eq(1)
before and after the post request.

On Thu, Feb 26, 2015 at 6:16 AM, Eliot Sykes <notifications@github.com>
wrote:

> In spec/requests/create_bookmark_from_email_spec.rb
> <https://github.com/LuisMenjivar/luisbookmark/pull/5#discussion_r25419727>
> :
>
> > @@ -13,12 +13,23 @@
> >      expect(@user.topics.first.bookmarks).to match_array [Bookmark.find_by(url: "www.google.com")]
> >      expect(response.status).to eq(200)
> >    end
> > -  xit "creates a bookmark if topic already exists" do
> > +  it "creates a bookmark if topic already exists" do
> > +    topic = create(:topic, title: "animals", user: @user)
> > +    post '/bookmarks',{sender: "test@example.com", subject: "animals", "body-plain": "www.google.com"}
> > +    expect(@user.topics.first.bookmarks.first).to belong_to(:topic)
>
> I think the belong_to(:topic) test here isn't testing what's intended.
> Check for yourself by commenting out line 17. The test will still pass
> without line 17 I suspect.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LuisMenjivar/luisbookmark/pull/5/files#r25419727>.
>